### DISCUSSION TITLE
Upgrade diff tool/website for react-native-tvos

### DISCUSSION THREAD

**Discussion by wouterds** [react-native-tvos/react-native-tvos#391] [General Discussions]
```
npx @react-native-community/cli upgrade 0.69.6-0
```

For example in last upgrade from **0.69.5-2** to **0.69.6-0** some things changed in the `AndroidManifest.xml` file. The tool also shows that it's not able to automatically determine the diff. The link below is to the upstream react-native project.

![Screen Shot 2022-11-10 at 09 32 16](https://user-images.githubusercontent.com/1210628/201040872-ee84ac97-e8f2-4cce-95a7-d1628e7fe3e1.jpg)

So what I usually do is remove the tvos suffix from the versions to compare. But obviously I now missed the tvos specific changes in `AndroidManifest.xml`. That's not ideal.

**Comment by douglowder** [comment]
I just created a new project with

```
react-native init TestUpgrade --template=react-native-tvos@0.69.5-2
```

and then did the upgrade -- there was no issue.

The upgrade helper web app does not work for the TV repo, it is configured for the RN core repo.

If you have a repo that is at 0.69.5-2 and will not upgrade successfully, please share that with me and I'll be happy to take a look.

**Reply by wouterds** [reply]
Yes I know it's specific to the upstream react-native lib, but that's what I want to highlight. It's a handy tool for upgrades to quickly compare diffs between versions without starting to dig into the repo and manually try to compare what changed. I'm not sure what it entails exactly to deploy something similar for react-native-tvos.

**Reply by douglowder** [reply]
I think it should just be a matter of publishing something in react-native-tvos.github.io that uses the correct URL https://github.com/react-native-tvos/rn-diff-purge-tv as the source of the diffs. If you could create an issue for that, I'll take a look 😄
